dataactvalidator/config/awardFinancialRules.csv

```diff
@@ -1,27 +1,27 @@
-MIN_LENGTH,TRUE,ObjectClass,4,,"Checks if object class is 4 digits, if so will check consistency with direct reimbursable flag",B11_C2,prerequisite
-CHECK_PREFIX,TRUE,ObjectClass,ByDirectReimbursableFundingSource,"{""1"":""D"",""2"":""R""}",Check that 1 digit prefix is consistent with value for ByDirectReimbursableFundingSource,B11_C3,prerequisite
-RULE_IF,TRUE,ObjectClass,B11_C3,B11_C2,"If object class is 4 digits, check that 1 digit prefix is consistent with value for ByDirectReimbursableFundingSource",B11_C4,file_validation
-SUM_FIELDS,FALSE,,DeobligationsRecoveriesRefundsOfPriorYearByAward_CPE,"ussgl487100_downwardadjustmentsofprioryearunpaidundeliveredordersobligationsrecoveries_cpe, ussgl487200_downwardadjustmentsofprioryearprepaidadvancedundeliveredordersobligationsrefundscollected_cpe, ussgl497100_downwardadjustmentsofprioryearunpaiddeliveredordersobligationsrecoveries_cpe, ussgl497200_downwardadjustmentsofprioryearpaiddeliveredordersobligationsrefundscollected_cpe",DeobligationsRecoveriesRefundsOfPriorYearByAward_CPE = USSGL(4871+ 4872 + 4971 + 4972). This applies to the award level.,C18,multi_field
-SET_EXISTS_IN_TABLE,FALSE,,ProgramActivity,"{""beginningperiodofavailability"":{""target_field"":""budget_year"",""skip_if_below"":2016}, ""agencyidentifier"":{""target_field"":""agency_id"",""pad_to_length"":3}, ""allocationtransferagencyidentifier"":{""target_field"":""allocation_transfer_id"",""pad_to_length"":3},""mainaccountcode"":{""target_field"":""account_number"",""pad_to_length"":4},""programactivitycode"":{""target_field"":""program_activity_code"",""pad_to_length"":4,""skip_if_blank"":true},""programactivityname"":{""target_field"":""program_activity_name"",""skip_if_blank"":true}}",Must be a valid program activity name and code for the corresponding TAS/TAFS as defined in Section 82 of OMB Circular A-11.,B9,multi_field
+rule_type,is_single_field,field_name,rule_text_one,rule_text_two,description,rule_label,rule_timing,original_label
+car_match,FALSE,,"allocationtransferagencyidentifier, agencyidentifier, beginningperiodofavailability, endingperiodofavailability, availabilitytypecode, mainaccountcode, subaccountcode","allocation_transfer_agency, agency_identifier, beginning_period_of_availability, ending_period_of_availability, availability_type_code, main_account_code, sub_account_code",TAS fields must match CARS file,A1_C,multi_field,A1
+REQUIRE_ONE_OF_SET,FALSE,,"piid, fain, uri",,"Must have either a piid, fain, or uri",NONE,multi_field,
```

Comment:
Labeling this as C14, will add the difference here to the spreadsheet for discussion with schema team.